test/src/TestDriver.cpp

```diff
@@ -1494,8 +1494,11 @@ TestACD()
   nmea_info.Reset();
   nmea_info.clock = TimeStamp{FloatDuration{1}};
 
-  /* $PAAVS responses from XPDR must be ignored */
-  ok1(!device->ParseNMEA("$PAAVS,XPDR,7000,1,0,1697,0,0*68",nmea_info));
+  /* test XPDR response */
+  ok1(device->ParseNMEA("$PAAVS,XPDR,7000,1,0,1697,0,0*68",nmea_info));
+  ok1(nmea_info.settings.has_transponder_code);
+  ok1(equals(nmea_info.settings.transponder_code.GetCode(), 
+             TransponderCode{07000}.GetCode()));
 
```

Comment:
Would be great to have tests with malformed/bad transponder values here, to ensure that error handling in the driver keeps working properly.

Reply:
added test to check against bogus value